pkg/controller/controller_instance.go

```diff
@@ -2381,6 +2381,40 @@ func (c *controller) prepareDeprovisionRequest(instance *v1beta1.ServiceInstance
 			return nil, nil, stderrors.New("InProgressProperties must be set when there is an operation or orphan mitigation in progress")
 		}
 		rh.inProgressProperties = instance.Status.InProgressProperties
+	} else if instance.Status.CurrentOperation == "" && instance.Status.ProvisionStatus != v1beta1.ServiceInstanceProvisionStatusProvisioned {
```

Comment:
What if `instance.Status.CurrentOperation != ""`, but the instance was never successfully provisioned? then it will still fail in the `else` condition below, since `ExternalProperties` will be nil?

Reply:
Actually you don't need to check `instance.Status.CurrentOperation` here, we have already checked it in the condition above:
```go
if instance.Status.CurrentOperation != "" || instance.Status.OrphanMitigationInProgress {
```

Reply:
sure, can skip CurrentOperation check